File: 6. semestr/[IBT] Bakalářská práce/Implementace/frontend/src/app/models/relationships/familyTreeRelationship.ts
/*******************************************************************/
/*  Project: Bachelor's thesis – Genealogy Database System         */
/*           Faculty of Information Technology                     */
/*           Brno University of Technology                         */
/*                                                                 */
/*  File:    familyTreeRelationship.ts                             */
/*  Date:    2019 – 2020                                           */
/*  Author:  Martin Chládek <xchlad16@stud.fit.vutbr.cz>           */
/*******************************************************************/

import { FamilyTree } from '../familytree';
import { Relationship } from '../relationship';

export interface FamilyTreeRelationship {

    familyTreeId: number;
    familyTree: FamilyTree;

    relationshipId: number;
    relationship: Relationship;
}

File: 6. semestr/[IBT] Bakalářská práce/Implementace/backend/Entity/Relationships/FamilyTreeRelationship.cs
/*******************************************************************/
/*  Project: Bachelor's thesis – Genealogy Database System         */
/*           Faculty of Information Technology                     */
/*           Brno University of Technology                         */
/*                                                                 */
/*  File:    FamilyTreeRelationship.cs                             */
/*  Date:    2019 – 2020                                           */
/*  Author:  Martin Chládek <xchlad16@stud.fit.vutbr.cz>           */
/*******************************************************************/

using System.ComponentModel.DataAnnotations.Schema;

namespace Entity.Relationships
{
    [Table("FamilyTreeRelationship")]
    public class FamilyTreeRelationship
    {
        public int FamilyTreeId { get; set; }
        public FamilyTree FamilyTree { get; set; }

        public int RelationshipId { get; set; }
        public Relationship Relationship { get; set; }
    }
}
